test/unit_test/character_sheet_unit_tests/FocusUnitTests.java

```diff
@@ -0,0 +1,30 @@
+package unit_test.character_sheet_unit_tests;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import rpg_database.character_sheet.Focuses;
+import rpg_database.character_sheet.Focus;
+
+public class FocusUnitTests {
+
+	@Test
+	public void testGetFocus() {
+		Focus expected = new Focus(Focuses.COURAGE);
+		assertEquals(Focuses.COURAGE, expected.getFocus());
+	}
+
+	@Test
+	public void getFocuseValue() {
+		Focus expected = new Focus(Focuses.JUMPING);
+		assertEquals(2, expected.getFocuseValue());
+	}
+
+	@Test
+	public void testSetFocusImprovement() {
+		Focus expected = new Focus(Focuses.JUMPING);
+		expected.MakeFocusImproved();
+		assertEquals(true, expected.isFocusImproved());

```

Comment:
"és egy külön tesztben megnézni hogy az isFocusImproved is jól működik -e" írtam feljebb. Ha így egymás után egy tesztbe két assertet raksz az nagyon gáz, mert ha az első elfailel, akkor nem látod tőle hogy a másodikkal mi történik, mert ugye ott kiugrik a teszt. Ez bad practice, csinálj rá két külön tesztet légyszi.

Bővebben: Ha a 40-es sorban elfailel az asserted akkor a 41-esben lévő sosem fut le.